runtime/tests/runtime_integration_tests/test_lms.rs

```diff
@@ -838,7 +838,7 @@ fn test_lms_verify_cmd() {
 
 #[test]
 fn test_lms_verify_failure() {
-    let mut model = run_rt_test(None, None, None);
+    let mut model = run_rt_test(RuntimeTestArgs::default());
```

Comment:
Also moving this refactor to a separate PR might be a good idea. Since the CSR change touches ROM, I think it would be good to keep this PR a bit smaller.

Reply:
Opened https://github.com/chipsalliance/caliptra-sw/pull/1776